### PR TITLE
VIH-7118 Buttons layout for wr judge

### PR DESCRIPTION
**Pull request by valentinabennett**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-7118

### Change description ###
 The button consultation made disabled until the connection is established and fix the layout of the buttons for iPad portrait view.

**Does this PR introduce a breaking change?** (check one with "x")

```
[X ] Yes
[ ] No
```

### CODE REVIEW

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.scss [hunk 1]
```diff
@@ -38,3 +38,12 @@
   max-height: 85vh;
   z-index: 100;
 }
+
+.vh-btn-text {
+}
+
+@media screen and (max-width: 1024px) {
```
Comment (shaed-parkar) [inline VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.scss:45]: I think you need to use screen only to make sure normal devices aren't affected by this change. This page has a list of combinations that affect tablets only. I hope this helps.
[css device queries](https://css-tricks.com/snippets/css/media-queries-for-standard-devices/#ipad-queries)

**Review by shaed-parkar — COMMENTED** [review]
Maybe we can create some functions or helpers in scss to determine if the browser is an ipad

File: VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.scss (hunk 1 above)
Comment (valentinabennett) [reply VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/judge-waiting-room/judge-waiting-room.component.scss]: I added 'only'
The fix should be ok for normal devices as it had not the correct layout if you resize the view, now it's should be better.

**Review by shaed-parkar — APPROVED** [review]
Please demo with the QA team before merging.

**Pull request merged**